File: 백준/Bronze/4153. 직각삼각형/직각삼각형.cs
while(true){
    int[] triangleArr = new int[3];
    triangleArr = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
    Array.Sort(triangleArr);
    if(triangleArr[0] == 0)
        break;
    else if((Math.Pow(triangleArr[0], 2) + Math.Pow(triangleArr[1], 2)) == Math.Pow(triangleArr[2], 2))
        Console.WriteLine("right");
    else
        Console.WriteLine("wrong");
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.cc
// 4153. 직각 삼각형

#include <iostream>
#include <cmath>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    while (true)
    {
        vector<double> sides(3);
        cin >> sides[0] >> sides[1] >> sides[2];
        sort(sides.begin(), sides.end(), [](double a, double b)
             { return a > b; });

        if (sides[0] == 0)
            break;
        else if (pow(sides[0], 2) == pow(sides[1], 2) + pow(sides[2], 2))
            cout << "right" << '\n';
        else
            cout << "wrong" << '\n';
    }
}